src/assets/svg/Linkedin.jsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+
+const Linkedin = ({ height, fill }) => (
```

Comment:
suuuuuuuuuper nit picky, but can this file and variable name be changed to `LinkedIn`?